homeassistant/components/sensor/deconz.py

```diff
@@ -113,6 +115,8 @@ def device_state_attributes(self):
         if self.unit_of_measurement == 'Watts':
             attr[ATTR_CURRENT] = self._sensor.current
             attr[ATTR_VOLTAGE] = self._sensor.voltage
+        if self._sensor.sensor_class == 'daylight':
+            attr[ATTR_DAYLIGHT] = self._sensor.daylight
```

Comment:
I don't see the need for this since it can be extrapolated from the state.

What I'm thinking about is if we shouldn't keep it as a binary sensor first, to follow the overall way the sensors work, e.g. name equals main state. Then we could have the resolved state as an attribute.

Reply:
Yeah it could go either way. To me it’s more useful at first glance to have the status string as the main state rather than daylight true/false since it’s the actually more useful part of the sensor. If the string was an attribute I’d just make a template sensor for it. But I see your point. 

It’s up to you, if you want it as a binary sensor I’ll make the necessary mods to pydeconz and here as well.

Reply:
You're right. Let's keep the useful information most easily accessible.

Reply:
Re: the "daylight" true/false attr; maybe we should keep it? It can be extrapolated by the user but as is it tracks precisely the interpretation deCONZ uses.

Reply:
Sure. Let's keep it

Reply:
OK. Great! Then I think this is ready to go if you’re otherwise happy. I addressed your other comment with a commit earlier this evening. 

PS this is another deconz device (like light groups) without a unique id so no entry in entity registry; I don’t think that matters, but the name of the entity created is quite generic (sensor.daylight).